dev/FHIR4-0-1-Test/_LoadResources/MP9-3-0-0-beta-load-resources-purgecreateupdate-xml.xml

```diff
@@ -35,14 +35,21 @@
          <reference value="../MP9-3-0-0-beta/_reference/fixtures/nl-core-HPrv-mp9-2165281100733-00001111.xml"/>
       </resource>
    </fixture>
+   <fixture id="Location-nl-core-HPrv-mp9-2165281100733-00005111">
```

Comment:
Do you know why new fixtures are being added to the LoadResources script? I wouldn't expect any differences here based on the change that has been done in src.